internal/compiler-bridge/src/main/scala/xsbt/ExtractAPI.scala

```diff
@@ -173,10 +173,10 @@ class ExtractAPI[GlobalType <: Global](
 
   private def thisPath(sym: Symbol) = path(pathComponents(sym, Constants.thisPath :: Nil))
   private def path(components: List[PathComponent]) =
-    new xsbti.api.Path(components.toArray[PathComponent])
+    xsbti.api.Path.make(components.toArray[PathComponent])
```

Comment:
I am not really liking the `make` here, so I am thinking about generating both `create` and `of`.
I wish I could use `apply` but Scala won't do it because `xsbti.api.Path` is not a value.